kod/object/item/passitem/ring.kod

```diff
@@ -63,14 +78,25 @@ messages:
 
    RevealHiddenAttributes()
    {
+      local i, bDone;
+
+      bDone = FALSE;
+      for i in plItem_Attributes
```

Comment:
This loop could use a comment.  It's not obvious what's going on here.

Reply:
Done.